ess-dmsc/python-nexus-utilities
# Add tool to output OFF file of full instrument geometry

Added `nexustooff.py` containing the function `nexus_geometry_to_off_file()`. This function takes a input NeXus filename and output OFF filename. It converts the entire geometry described in the NeXus file to a single OFF file. The OFF file can then be viewed using `render_off_from_file()` from `drawoff.py`.

**To test:**
Run `SANS2D_example.py`, followed by `nexustooff.py`, which will generate a NeXus file then convert that to an OFF which will then be rendered in a 3D view. You should see the Utah teapot in the sample position and the two detector panels of the SANS2D instrument.
